pkgs/development/python-modules/qutip/default.nix

```
@@ -0,0 +1,57 @@
+{ lib, fetchFromGitHub, python3Packages }:
```

```suggestion
{ lib, fetchFromGitHub, buildPythonPackage, packaging, numpy, cython, scipy, matplotlib, pytest, pytest-rerunfailures }:
```
Passing the entire python package set as dependencies make the override mechanism more difficult to use, e.g. you can't simpy 
```
qutip = qutip.override { numpy = myNumpy };
```

---

> Passing the entire python package set as dependencies make the override mechanism more difficult to use, e.g. you can't simpy

Not only that but it breaks splicing and is strictly forbidden.

---

Ok, sorry, didn't have that on my radar. 🙈